sklearn/gaussian_process/_gpr.py

```diff
@@ -198,8 +198,12 @@ def fit(self, X, y):
             self._y_train_std = np.std(y, axis=0)
 
             # Remove mean and make unit variance
-            y = (y - self._y_train_mean) / self._y_train_std
-
+            # Moreover, add a very small number to the y_train.std to
+            # avoid a divide by zero error.
+            if self._y_train_std.all() == 0:
```

Comment:
Indeed, I forgot you may have multiple targets: you might want to add epsilon only to target having zero std.